cmd/commands/env.go

```diff
@@ -106,16 +107,21 @@ func envCmd(ctx *cli.Context) error {
 
 		// Takes the complement of previousPaths and sdkPaths, and removes the complement from osPaths.
 		previousPaths := env.NewPaths(env.PreviousPaths)
+		removeCnt := 0
```

Comment:
```suggestion
		jointCnt := 0
```
`removeCnt` may not be accurate, `jointCnt` is better.

Reply:
already edited